src/ArgumentFormatter.php

```diff
@@ -23,7 +23,7 @@ public function format(array $arguments, bool $recursive = true): string
         foreach ($arguments as $argument) {
             switch (true) {
                 case is_string($argument):
-                    $result[] = '"' . $argument . '"';
+                    $result[] = '"' . substr($argument, 0, 1000) . '"';
```

Comment:
I wonder if it's worth adding an ellipsis here to emphasise that it's been cut off. For example:
```suggestion
                    $result[] = '"' . substr($argument, 0, 1000) . '..."';
```

Not sure if it should be 997 characters, plus the ellipsis... or 1000 plus the ellipsis.

Reply:
Good point, updated this PR with those changes!